triangle.json

```diff
@@ -0,0 +1,98 @@
+{
+    "#": [
+        "Pursuant to discussion in #202,",
+        "we have decided NOT to test triangles where a + b = c.",
+        "e.g: (2, 4, 2, Isosceles), (1, 3, 4, Scalene).",
```

Comment:
I think we had already found a definitive reason in prior discussions, regarding degenerative triangles.  Doesn't 0 as a any length create a degenerative triangle, and therefore is rejected on that basis?

I think we have decided to test, and decided to fail them, so the sentence there does not make sense.